### DISCUSSION TITLE
<Your question or feedback about the GitHub Discussions product>

### DISCUSSION THREAD

**Discussion by btallydocscom** [community/community#126116] [GitHub Education]
### Select Topic Area

Question

### Body

I am not able to submit the form as it keeps on telling not in the campus.

1. My university location on google map is * litebrite82 removed Personal Identifying Information. 
2. I stay in the hostel at * litebrite82 removed Personal Identifying Information 

I have raised a support ticket #2786731 no response for last two weeks.

**Comment by LiteBrite82** [comment]
Hi @btallydocscom ,

I've removed your private information from [this post](https://github.com/orgs/community/discussions/126116#discussion-6740620) as posting identifiable information is not allowed, per our [Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md).

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
